stellargraph/layer/gcn.py

```diff
@@ -211,10 +200,6 @@ def call(self, inputs):
             output += self.bias
         output = self.activation(output)
 
-        # On the final layer we gather the nodes referenced by the indices
-        if self.final_layer:
-            output = K.gather(output, out_indices)
-
         # Add batch dimension back if we removed it
         # print("BATCH DIM:", batch_dim)
         if batch_dim == 1:
```

Comment:
Same as before, I don't think we need to check that `batch_dim==1` here.

Reply:
For this one, the `batch_dim == 1` special case is removed entirely in #1205.